BlinkID/lib/recognizers/blink_id_multi_side_recognizer.dart

```diff
@@ -181,8 +181,11 @@ class BlinkIdMultiSideRecognizerResult extends RecognizerResult {
   ///image of the signature if enabled with returnSignatureImage property.
   String? signatureImage;
 
+  Map<String, dynamic>? nativeResult;
```

Comment:
Maybe add it to the super class - `RecognizerResult`? So that all recognizes would benefit?

Reply:
Done. I had to re-run the builder.